server/src/main/java/org/opentosca/toscana/plugins/cloudfoundry/CloudFoundryLifecycle.java

```diff
@@ -0,0 +1,40 @@
+package org.opentosca.toscana.plugins.cloudfoundry;
+
+import java.io.IOException;
+
+import org.opentosca.toscana.core.transformation.TransformationContext;
+import org.opentosca.toscana.plugins.lifecycle.AbstractLifecycle;
+
+import org.slf4j.Logger;
+
+public class CloudFoundryLifecycle extends AbstractLifecycle {
+
+    private final TransformationContext context;
+    private final Logger logger = super.logger;
+
+    public CloudFoundryLifecycle(TransformationContext context) throws IOException{
+        super(context);
+        this.context=context;
```

Comment:
Please reformat your code using `CTRL + ALT + L` (on mac propbably `CMD + ALT + L`) before commiting.

According to the default checkstyle configuration this should be:
`this.context = context;`